alf/algorithms/entropy_target_algorithm.py

```diff
@@ -35,42 +36,64 @@ class EntropyTargetAlgorithm(Algorithm):
     It tries to adjust the entropy regularization (i.e. alpha) so that the
     the entropy is not smaller than `target_entropy`.
 
-    It's described in section 5 of the following paper:
-    Haarnoja et al "Soft Actor-Critic Algorithms and Applications" arXiv:1812.05905v2
+    The algorithm has two stages:
+    1. init stage. During this stage, the alpha is not changed. It transitions
+       to fast_stage once entropy drops below `target_entropy`.
+    2. adjust stage. During this stage, log_alpha is adjusted using this formula:
+       ((above + 0.5 * below) * increasing - (below + 0.5 * above) * decreasing) * update_rate.
+       Note that log_alpha will always be increased if entropy is increasing
```

Comment:
Explanation inconsistent with the formula. From the formula, log_alpha will always be increased if the entropy is increasing (whether it's above or below the target), otherwise it will always be decreased.

Reply:
The comment does not contradict with the formula. It says log_alpha will increase even in the situation of the entropy being below the target as long as the entropy is increasing.

Reply:
I mean for the sentence "Same reason for always decreasing log_alpha when the entropy is above the target entropy."

Reply:
And "even when the entropy is below the target entropy" should be "even when the entropy is above the target" because usually people would expect that if the entropy is above the target, alpha should be decreased. Here you are using the word "even when" to represent a different situation from the common sense.

Reply:
You mean "even in the situation of the entropy being above the target"? Because if it's below, log_alpha is expected to increase (so no need to use the word "even"). Or are you talking about the "entropy loss" (negative) instead of "entropy" (positive)?